### PR TITLE
`React.createClass` is deprecated and will be removed in version 16

### PR DESCRIPTION
**Pull request by klaasman**
React provides a drop-in replacement `create-react-class`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,6 +21,7 @@
   },
   "license": "MIT",
   "peerDependencies": {
+    "create-react-class": "^15.5.0",
```
Comment (klaasman) [inline package.json:24]: I'm not sure whether it should be a `dependency` or `peerDependency`

**Comment by NeXTs** [comment]
@Lapple Hey Aziz, please spend a minute to accept and publish this PR.

It's quite important for those of us who want to migrate to react v16

**Comment by soneymathew** [comment]
thanks for this https://www.npmjs.com/package/react-transitive-number-v16

**Comment by Lapple** [comment]
Hi, @NeXTs, @soneymathew, @klaasman, my bad for leaving this for so long. Will release a new minor in a minute.

**Comment by klaasman** [comment]
It has only been 5 months, no worries ;-)

Did you see my question about the peer dependency?

**Comment by Lapple** [comment]
😄 things move slow in unmaintained projects. I think it works either way, npm would flatten the `dependency` into peer should it be used in multiple React plugins, right?

**Comment by klaasman** [comment]
Well, I'm not really familiair with the npm ecosystem, hence the question ;-)

My understanding of a peer dependency is that it isn't automatically installed, but should be installed by its dependent.

**Pull request merged**